scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
The zlib case was just to be explicit. One of the others (which one escapes me now) tried to use "gcc" for the build tools. In response I added that to all of the scripts in companion libs. "gcc" doesn't exist by default on FreeBSD (cc is clang, there is no gcc), so it failed.